src/services/assets.js

```diff
@@ -410,3 +415,26 @@ export async function calculateGasEstimate(transaction: Object) {
     )
     .catch(() => DEFAULT_GAS_LIMIT);
```

Comment:
Spotted another here:
`Sentry.captureException(err);`

Reply:
For existing ones I would say create another ticket as it's massive work.